runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -377,6 +378,22 @@ public void windowClosing(WindowEvent event)
 					}
 				}
 			});
+
+			frame.addWindowFocusListener(new WindowFocusListener()
```

Comment:
this is registered before keyListeners is setup, is that a problem?

Reply:
if its initially null rather than empty it could, I've fixed that in the recent commit as it makes more sense regardless